README.md

```diff
@@ -21,15 +21,48 @@ xpip install xontrib-hist-navigator
 xontrib load hist_navigator
 ```
 
+# Usage
+
+1. Add the following to your `.py` xontrib loading config and `import` it in your xonsh run control file (`~/.xonshrc` or `~/.config/rc.xsh`):
+```py
+from xonsh.xontribs 	import xontribs_load
+from xonsh.built_ins	import XSH
+envx = XSH.env
+
+xontribs = [ "hist_navigator", # Initializes hist_navigator (fish-shell-like dir history navigation)
+ # your other xontribs
+]
+# ↓ optional configuration variables (use `False` to disable a keybind)
+if 'hist_navigator' in xontribs: # configure xontrib only if it's loaded
+  # config var                 value  |default|alt_cmd¦ comment
+  envx["X_HISTNAV_KEY_PREV"] = "⎇←"  #|['escape','left' ]|False¦ Move to the previous working directory
```

Comment:
It is better to name variables as `XSH_*` , I have some other xontribs using this form.